### DISCUSSION TITLE
Integration with Video Editor

### DISCUSSION THREAD

**Discussion by anciltech** [mifi/lossless-cut#1188] [Ideas]
Hi @mifi 

Great work on the LC and all your other projects! 

Have you considered tying what you've built here into major video editing programs like Final Cut? Looks like functionality like in [Worx4](https://apps.apple.com/us/app/worx4-x/id1195903030?ls=1&mt=12) would be right up your ally, and provide some automation to the entire trimming process based off of the tags/selections people have made in their editing programs.

Cheers
Max

**Comment by mifi** [comment]
Hi. Do I undedstand it correctly, that it takes an existing fcp project, then cuts all the source video files to remove all footage that is not currently in use by according to the timeline/project?

**Reply by anciltech** [reply]
Yes, exactly. I think it might also look at clips/sections labeled as keepers/rejects in the fcx library and use this information as well.

**Reply by mifi** [reply]
I think it's a cool idea, but TBH I don't see where it fits in to losslesscut. The way I see it, it's really more like a separate product which would basically be a plugin for FCP and such editors.
